src/pages/home/report/ReportActionsView.js

```diff
@@ -120,6 +127,13 @@ class ReportActionsView extends React.Component {
     }
 
     shouldComponentUpdate(nextProps, nextState) {
+        const policy = this.props.policies[`${ONYXKEYS.COLLECTION.POLICY}${this.props.report.policyID}`];
+        const nextPolicy = nextProps.policies[`${ONYXKEYS.COLLECTION.POLICY}${nextProps.report.policyID}`];
+
+        if (!_.isEqual(policy, nextPolicy)) {
```

Comment:
Shouldn't avatar equality be enough? Policy will have other values for which we might not want to reload.

Reply:
@hungvu193